src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.BuildOrder.targets

```diff
@@ -20,13 +20,20 @@ properties that determine build ordering.
       _CreatePropertiesCache;
       _CleanIntermediateIfNeeded;
       _CheckProjectItems;
+      AndroidPrepareForBuild;
```

Comment:
This target runs tooling resolution (`$(_OnResolveMonoAndroidSdks)`), and sets up properties and paths related to the Android SDK, JDK, and others. I was thinking about trying to move this earlier in the list since we seem to have a lot of targets that depend on it which should cause it to run earlier anyway. I'm not sure that's worth the potential risk at this time though, thoughts?